feat: FAQ - Add Help Center (#571)
lukso-network/docs: docs/faq/help-center.md

> ---

Reviewer: I'm not sure it is necessary to change the file name (which impacts the URL).

If you do change a page URL, I highly recommend to add a redirect:

https://github.com/lukso-network/docs/blob/main/docusaurus.config.js#L19

This is for SEO purposes and also to make sure all exising links to this page don't go to a 404.

But i think we can keep "help.md" (so `/help` URL)

Reviewer 2: Agree on this

sidebar_label: 'Help Center'
sidebar_position: 5
---

# Help Center

> ### How to stay up to date about LUKSO?

Reviewer: why go from h1 to h3 directly without h2?

Reviewer 2: Yes let's keep h2 so ##, so that we also have a good ToC on the right sidebar

> You can follow us on [Twitter](https://twitter.com/lukso_io), read our [Medium](https://medium.com/lukso) articles, and follow the announcements on our [Discord](https://discord.gg/lukso) server.

Reviewer 2: Suggesting to not use "our" but keep it general.

```suggestion
You can follow LUKSO on [Twitter](https://twitter.com/lukso_io), read the latest article on [Medium](https://medium.com/lukso) articles, and follow the announcements on the LUKSO [Discord](https://discord.gg/lukso) server.
```

> ### Where can I read LUKSOs Whitepaper?

Reviewer 2: ```suggestion
### Where can I read the LUKSO Whitepaper?
```

Here is the link to read and download the [Official LUKSO Whitepaper](https://uploads-ssl.webflow.com/629f44560745074760731ba4/62b200bfe0af12186845519a_LUKSO_Whitepaper_V1-1.pdf).

### My question is not here. Where can I ask it?

> If you can't find an answer to your question, you can head to our [Discord](https://discord.gg/lukso) server. We have development-related channels where you can ask technical questions. We will add the most asked question to this list.

Reviewer 2: Might be worth mentioning the channels here, related to specific topics?

### Where can I report regular issues for repositories and tools?

> We are preparing workflows to handle issues, investigations, and bug bounties. For now, please create an issue on our global [GitHub Issue Tracker](https://github.com/lukso-network/issue-tracker).

Reviewer 2: Yes we should encourage people to post issues in the actual repos.

For the browser extension, I would suggest the issue tracker link mentioned here.

### Where can security issues be reported to?

> Suppose it's a vulnerability or something that should not be shared with the public. Please reach out to official team members within our developer channel on [Discord](https://discord.gg/lukso).

Reviewer 2: We should suggest that they reach out to the LUKSO team on these server, send us a friend request and let us know by DM.

### Does LUKSO have grants or bug bounties?

> We're happy to compensate if there are severe findings and vulnerabilities discovered. The compensation will depend on the scope of the error or bug. We do not share precise bounty amounts beforehand because of the wide range of possible discoveries.

Reviewer 2: We need to check this info with the business team before hand.
Please check with Jonathan.

Author: I have checked this with Fabian directly
